Tentacle/Client.swift

```diff
@@ -256,7 +270,16 @@ public final class Client {
     public func userWithLogin(login: String) -> SignalProducer<(Response, UserInfo), Error> {
         return fetchOne(.UserInfo(login: login))
     }
-    
+
+    public func issues(page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Issue]), Error> {
+        return fetchMany(.Issues, page: page, pageSize: 30)
+    }
+
+    public func issuesForRepositories(repository: Repository, page: UInt = 1, perPage: UInt = 30) -> SignalProducer<(Response, [Issue]), Error> {
```

Comment:
A better name for this might be `issuesInRepository`. _in_ instead of _for_ and _repository_ singular.